projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -65,12 +81,7 @@ export default function SubscribePanels() {
 	const showNotices = Number.isFinite( subscriberCount ) && subscriberCount > 0;
 	return (
 		<>
-			{ isNewsletterFeatureEnabled() && (
-				<PluginDocumentSettingPanel title={ __( 'Newsletter', 'jetpack' ) }>
-					<NewsletterAccess setPostMeta={ setPostMeta } accessLevel={ accessLevel } />
-				</PluginDocumentSettingPanel>
-			) }
-
+			{ renderAccessLevelSelectorPanel( setPostMeta, accessLevel ) }
```

Comment:
I think the React way would be 
```suggestion
			<RenderAccessLevelSelectorPanel accessLevel={accessLevel}  onChange={ setPostMeta } />
```

Reply:
No. React way would be `AccessLevelSelectorPanel`

Reply:
Done!